clients/wallet/src/horizon/responses.rs

```diff
@@ -152,14 +193,45 @@ pub struct TransactionResponse {
 	pub memo: Option<Vec<u8>>,
 }
 
+impl Debug for TransactionResponse {
```

Comment:
the simple derive `debug` prints the value of the fields in `Vec<u8>`. Converting them to a readable string will make it easier when we want to view the xdr.